webapi/tct-extra-html5-tests/html5-extra-app/testscripts/get-navigatorlanguage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: system-state-and-capabilities
+ Scenario: get navigator language
+  When launch "html5-extra-app"
+   And switch to language "English (United Kingdom)"
+   And switch to "html5-extra-app"
+   And I go to "w3c/system-state-and-capabilities/the-navigator-object/bdd/get-navigatorlanguage-manual.html"
+  Then I should see "Pass" in 5 seconds
+   And switch to language "English (United States)"
```

Comment:
I think we should reset the language before assertion to avoid the situation that it couldn't be executed when previous step fails. 
You can save the result first.